app/views/contexts/summary/group.py

```diff
@@ -28,7 +28,11 @@ def _build_blocks(
         block_ids_on_path = [location.block_id for location in path]
 
         for block in group_schema['blocks']:
-            if block['id'] in block_ids_on_path and block['type'] == 'Question':
+            if (
+                block['id'] in block_ids_on_path
+                and block['type'] == 'Question'
+                and not block.get('question', {}).get('hide_on_section_summary', False)
```

Comment:
Does `hide_on_section_summary` need to live within `question`?. We won't have more than 1 question per block hence we can define it at the block level?. And this currently wouldn't work for `question_variants`?

Reply:
ARE you suggesting have this as an allowable key only on Question type blocks?

Reply:
I think that would make sense?. Defined at the block level and validator can ensure it is defined on allowed types.

Reply:
Yeah ok, it's a small change, and would extend the functionality to question_variants